SelectBox - Getting Started: Add ASP.NET Core Overhaul (#9074)
DevExpress/devextreme-documentation: concepts/05 UI Components/SelectBox/00 Getting Started with SelectBox/25 Group Data.md

                    type: 'array',
                    key: 'ID',
                },
                group: 'Category',
            });
        }
    }

##### Vue

    <!-- tab: App.vue -->
    <template>
        <DxSelectBox
            :data-source="dataSource"
            :grouped="true"
        />
    </template>

    <script setup lang="ts">
    import { DxSelectBox } from 'devextreme-vue/select-box';
    import { DataSource } from 'devextreme-vue/common/data';

    const dataSource = new DataSource({
        store: {
            data,
            type: 'array',
            key: 'ID',
        },
        group: 'Category',
    });
    </script>

##### React

    <!-- tab: App.tsx -->
    import { SelectBox } from 'devextreme-react/select-box';
    import { DataSource } from 'devextreme-react/common/data';

    const dataSource = new DataSource({
        store: {
            data: data,
            type: 'array',
            key: 'ID'
        },
        group: 'Category'
    })

    export default function App() {
        return (
            <SelectBox
                dataSource={dataSource}
                grouped={true}
            />
        );   
    }
    
---

> SelectBox can also display grouped data using nested data sources with only one nesting level. Each object in a nested data source contains a **key** field and an **items** array. For more information, refer to the following guide: [SelectBox - Grouping in the Data Source](/Documentation/Guide/UI_Components/SelectBox/Grouping/In_the_Data_Source/).

Reviewer: ```suggestion
SelectBox can also display grouped data from a nested data source with one nesting level. Each object in the data source contains a key field and an items array. For more information, refer to the following guide: [SelectBox - Grouping in the Data Source](/Documentation/Guide/UI_Components/SelectBox/Grouping/In_the_Data_Source/).
```
